docs/user/rstudio/ide/guide/tools/version-control.qmd

```diff
@@ -54,43 +54,43 @@ An excellent resource for learning more about Subversion and how to use it is th
 
 ## Getting Started
 
-Once you've installed your preferred Version Control system, you'll need to activate it on your system by following these steps:
+Once a Version Control system has been installed, it needs to be activated on the system by following these steps:
 
-1.  Go to **Global Options** (from the Tools menu)
+1.  From the **Tools** menu, click **Global Options**
 
-2.  Click **Git/SVN**
+2.  Click on the **Git/SVN** tab
 
 3.  Click **Enable version control interface for RStudio projects**
 
-4.  If necessary, enter the path for your Git or SVN executable where provided. You can also create or add your RSA key for SSH if necessary.
+4.  If necessary, enter the path for the Git or SVN executable where provided. A SSH key can be created or added for SSH if necessary.
+
+![](images/version-control-options.png){fig-alt="A screenshot of the Git/SVN tab in Global Options displaying an active Git executable."}
 
-![](images/version-control-options.png){fig-alt=""}
+If you are using RStudio Server, by default RStudio Server will look at the PATH for the first Git executable it finds, and sets that as the default. If you have a specific version that you want to set for all users, the easiest option is to make sure it's the first available on the PATH system-wide.
 
-If you are using RStudio Server, by default RStudio Server will look at the PATH for the first Git executable it finds, and sets that as the default. If you have a specific version that you want to set for all users, the easiest option is to make sure it's the first available on the PATH system-wide.\
-\
 Alternately, if you are running version 1.4 or above of Posit Workbench (previously RStudio Server Pro/RStudio Workbench), you can set the default Git version system-wide by setting the `git_exe_path` option in your system rstudio-prefs.json file. See here for more:\
-<https://docs.rstudio.com/ide/server-pro/r-sessions.html#customizing-session-settings>\
-<https://docs.rstudio.com/ide/server-pro/session-user-settings.html>
+<https://docs.posit.co/ide/server-pro/r-sessions.html#customizing-session-settings>\
+<https://docs.posit.co/ide/server-pro/session-user-settings.html>
```

Comment:
these two links didn't load for me. are they live yet?

Reply:
They are not but will be by the release of the IDE guide